ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -10916,6 +10916,9 @@ ABoundActivity * HqlCppTranslator::doBuildActivityOutput(BuildCtx & ctx, IHqlExp
     IHqlExpression * dataset  = expr->queryChild(0);
     IHqlExpression * rawFilename = queryRealChild(expr, 1);
 
+    // Determine if we should use generic read/write code
+    bool useGenericReadWrites = options.genericDiskReadWrites && targetHThor();
```

Comment:
No need for the hthor test - that should be already covered in hqlcpp(2032) above.

Reply:
This variable was removed entirely as it is no longer used.